### PR TITLE
Feat/add filters to service model

### PR DESCRIPTION
**Pull request by jcidp**
Adding Pet Type & Pet Number to the services. This considers:
- The filter
- Service creation
- Seeds
- Service view

### CODE REVIEW

File: pawsitive/db/migrate/20240518174648_create_messages.rb
Comment (lincoln1883) [inline pawsitive/db/migrate/20240518174648_create_messages.rb:1]: @jcidp, good job it works well but, this file should not be included in the pr i already have the corrected version pending.

Comment (jcidp) [reply pawsitive/db/migrate/20240518174648_create_messages.rb:1]: Done

**Review by lincoln1883 — APPROVED** [review]
@jcidp, great job on this ticket it time to merge.
